baseball/src/Components/Display/Display.js

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from "react";
+import Dashboard from "../Dashboard/Dashboard";
+
```

Comment:
Nice job composing your components, and making the `dashboard` a stateless 'presenter' component Stefan 🔥